src/index.ts

```diff
@@ -0,0 +1,21 @@
+import { Observable, of } from 'rxjs'
+import { Proposal } from './proposal'
+import { Operation } from './operation'
+import { DAO } from './dao'
+
+export default class Arc {
+  constructor(private graphqlProvider: string, private web3Provider: string) {}
+
+  daos(): Observable<DAO[]> {
+    throw new Error('not implmented')
+  }
+
+  dao(address: string): DAO {
+    return new DAO(address)
+  }
+
+  /**
+   * An observable of the list of pending operations
```

Comment:
ok. This would be all the `Sent` transactions currently running and it should also be persisted through browser refresh. 
We should also add an `operationStarted`/`operationFinished` observables.